charts/eks-anywhere-packages/templates/cronjob.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  namespace: {{ .Values.namespace }}
+  name: cron-ecr-renew
+spec:
+  schedule: "0 */6 * * *"
+  successfulJobsHistoryLimit: 3
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: OnFailure
+          serviceAccountName: {{ include "eks-anywhere-packages.serviceAccountName" . }}
```

Comment:
Should this have a different service account with different K8s rbac since our actual package controller doesn't have the same requirements to write configmaps/secrets? I would be ok with this for now, since this is temporary even though it's the best.

Reply:
I believe the package controller already has ability to write configmaps. As for secrets the package controller here only consumes secrets. Helm is actually the one that is creating the secret in other namespaces via passing in the value to the helm charts